stories/virtru/sendMessage/basic.jsx

```diff
@@ -96,26 +97,32 @@ storiesOf('virtru/sendMessage', module).add('basic', () => {
                     </Col>
                   </Row>
                   {fileList.length > 0 && (
-                    <Row gutter={[12, 12]}>
-                      {fileList.map((file) => {
-                        return (
-                          <Col key={file.name} md={12}>
-                            <FileListItem
-                              name={file.name.split('.')[0]}
-                              extension={`.${file.name.split('.')[1]}.tdf`}
-                              decryptedExt={`${file.name.split('.')[1]}`}
-                              size="1.09 Kb"
-                              tooltipText="This file is encrypted with no additional security options enabled."
-                              warning={false}
-                              button={
-                                <Button type="primary" size="small" onClick={handleClickRemove}>
-                                  Remove
-                                </Button>
-                              }
-                            />
-                          </Col>
-                        );
-                      })}
+                    <Row gutter={[12, 12]} id="itemsList">
+                      <ArrowsNavigationWrap
+                        tabbingSelector="[data-testid='item-wrapper']"
+                        parentSelector="#itemsList"
```

Comment:
We still do not want to do this. Going upwards in the DOM is against React's ideals. We want all data to flow one-way, so we must think about how this component can be the parent. Then, this component will only search inside itself, not upwards in the DOM.

To do that, you may need to use [React refs](https://reactjs.org/docs/refs-and-the-dom.html). These will allow you to hold on to the proper element without needing to query the DOM. This is what React uses internally, and should be used within our components. Manual DOM querying should be very, very rare.

I also previously mentioned [Render props](https://reactjs.org/docs/render-props.html). You may need to use render props in order to create the ref inside `ArrowsNavigationWrap`, and pass it to the `Row` to use.